src/sysext/sysext.c

```diff
@@ -0,0 +1,987 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <fcntl.h>
+#include <getopt.h>
+#include <sys/mount.h>
+#include <unistd.h>
+
+#include "capability-util.h"
+#include "dissect-image.h"
+#include "escape.h"
+#include "fd-util.h"
+#include "fileio.h"
+#include "format-table.h"
+#include "fs-util.h"
+#include "hashmap.h"
+#include "log.h"
+#include "machine-image.h"
+#include "main-func.h"
+#include "missing_magic.h"
+#include "mkdir.h"
+#include "mount-util.h"
+#include "mountpoint-util.h"
+#include "os-util.h"
+#include "pager.h"
+#include "parse-util.h"
+#include "pretty-print.h"
+#include "process-util.h"
+#include "sort-util.h"
+#include "stat-util.h"
+#include "terminal-util.h"
+#include "user-util.h"
+
+static enum {
+        ACTION_STATUS,
+        ACTION_MERGE,
+        ACTION_UNMERGE,
+        ACTION_REFRESH,
+        ACTION_LIST,
+} arg_action = ACTION_STATUS;
```

Comment:
I'd like to use `Verb`. That is, `systemd-sysext merge` instead of `--merge`.

Reply:
So the reason this used simple options was that I wanted to allow command lines such as `systemd-sysext --merge --unmerge` which combines the two operations in one. With verbs that couldn't be covered so nicely.

However, ultimately I later added `--refresh` as combined optoin for both, which kinda moots my thinking there.

So yeah, maybe, yes, this should be reworked to use verbs.

Reply:
This is now addresses in #18311